### ISSUE TITLE
there's unstable feature no_core

### ISSUE THREAD

**Issue by bingmatv** [phil-opp/blog_os#1358]
There's unstable feature
~~~
#![no_core]
~~~
Is it useful for developing OS?

**Comment by bjorn3** [comment]
No, not at all. It is completely useless for any code that a user of rust will write. `#![no_core]` is super unstable and requires defining a ton of language items and compiler intrinsics with any mistake potentially causing the compiler itself to crash. The only reason it exists at all is because libcore needs it to avoid depending on itself.

**Comment by ChocolateLoverRaj** [comment]
Can this issue be closed?
